packages/@aws-cdk/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -135,11 +149,18 @@ export interface NodegroupOptions {
   /**
    * The instance type to use for your node group. Currently, you can specify a single instance type for a node group.
    * The default value for this parameter is `t3.medium`. If you choose a GPU instance type, be sure to specify the
-   * `AL2_x86_64_GPU` with the amiType parameter.
+   * `AL2_x86_64_GPU` with the amiType parameter. This property will be ignored if `instanceTypes` is defined.
```

Comment:
It's better to throw an error if both `instanceType` and `instanceTypes` is used. Will make the deprecation easier since we won't have to maintain this dual scenario.